src/views/AddCommunity/index.tsx

```diff
@@ -161,17 +158,15 @@ const AddCommunity: React.FC<{ isLoading?: boolean }> = props => {
 
                         if (result?.status === 200) {
                             try {
-                                if (!signature) {
```

Comment:
The purpose of this check was, request the user to sign a message only if we don't have a signature on the store already, to avoid the user to keep signing messages every time, if the signature is stored and is valid, we don't need to ask the user to sign another message. Without this check here i believe, we're asking the user to sign a message everytime regardless if he already has a valid signature stored.

Reply:
i asked for the signature prompt to always be shown because if i'm in one account, with the signature already valid, and disconnect from that account, when i login in a new account it doesn't ask for a signature. it's like the signature is already signed but i signed it with the previous user. the variable signature is still signed even if I disconnect and connect with other accounts.
if I don't ask always for signature, only refreshing the page will solve this situation that i explained